src/actions/resize/IResizeTypes.ts

```diff
@@ -0,0 +1,16 @@
+import ResizeAction from "./ResizeAction";
+
+/**
+ * @memberOf Types
+ * @name resizeActionFactory
+ * @description
+ * This is the type for all resize action methods (scale, fit, crop, etc.).
+ */
+export type resizeActionFactory = (width?:number|string, height?:number|string) => ResizeAction
```

Comment:
Why the word "factory" do we use this for other builders?

Reply:
This is a good question.
Strictly speaking, these functions are factories that return a Builder Class for an action.

Resize has a lot of these functions, and they're all reused, so it made sense to extract all these declarations into one place.

We could create these types for all of our action functions, but if they don't share the signature then it makes no sense.

The idea was the share the signature of the resizeAction(factory)
What do you think?

Reply:
As long as it's internal only and does not affect the user im ok with it